bin/deploy-tile-server.ts

```diff
@@ -59,4 +62,38 @@ export function deployTileServer(
   // Establish a deployment dependency to ensure the tile server container stack
   // is fully deployed before initiating the deployment of the data plane stack.
   tsDataplaneStack.addDependency(tileServerContainerStack);
+
+  // Testing infrastructure
+  const imageryStack = new TSImageryStack(
+    app,
+    `${targetAccount.name}-TSImagery`,
+    {
+      env: targetEnv,
+      account: targetAccount,
+      vpc: vpcStack.resources.vpc,
+      description:
+        "Tile Server test imagery, Guidance for Overhead Imagery Inference on AWS (SO9240)"
+    }
+  );
+  imageryStack.addDependency(vpcStack);
+
+  const tsEndpoint: string = `http://${tsDataplaneStack.resources.fargateService.loadBalancer.loadBalancerDnsName}/latest`;
+  const tsTestImageryBucket: string =
+    imageryStack.resources.imageBucket.bucket.bucketName;
+  const tileServerTestRunnerStack = new TSTestRunnerStack(
+    app,
+    `${targetAccount.name}-TSTestRunner`,
+    {
+      env: targetEnv,
+      account: targetAccount,
+      osmlVpc: vpcStack.resources,
+      tsEndpoint: tsEndpoint,
+      tsTestImageBucket: tsTestImageryBucket,
+      buildFromSource: buildFromSource,
+      description:
+        "Tile Server test runner, Guidance for Overhead Imagery Inference on AWS (SO9240)"
```

Comment:
I am thinking, maybe in a separate PR. Anything that's inside a `model_runner` should be named as 

> Model Runner, Guidance for Overhead Imagery Inference on AWS (SO9240)

Then anything inside `tile_server` should be named as

> Tile Server, Guidance for Overhead Imagery Inference on AWS (SO9240)

It makes it clean and consistent.

Reply:
I was going to propose this myself!  I'll update this file to match that convention and then we can have another PR to align the rest of them.